### PR TITLE
DKG stats-dapp QA (Sept 13, 2023) fixes part 1

### LINKED ISSUE #1599: Add SEO for DKG stats-dapp & update subdomain
### Overview

- [ ] SEO Metadata: Update meta titles, descriptions, SEO image, and keywords to reflect Tangle's branding.
- [ ] URL Consideration: Incorporate 'Tangle' into the website URL for better clarity and branding. Note: we must audit all existing links and replace them appropriately to avoid broken links for the update.

### Assets
- Tangle SEO Image: https://www.notion.so/hicommonwealth/Tangle-SEO-Image-c7ca16709f86437cac818f2d05f3811e?pvs=4

### LINKED ISSUE #1632: Mismatch between authority ID value & display value on the key detail page (shouldn’t have 0x prefixed).
Untitled

### LINKED ISSUE #1633: Large empty space when there are few items in the table.

### LINKED ISSUE #1634: Add a link to the key detail page.
Untitled

### LINKED ISSUE #1635: Page's history feature is currently unsupported, so the history button should not be displayed.
https://github.com/webb-tools/webb-dapp/assets/53374218/51d6de81-5c5e-4f43-8a76-10b0560f8dea

### LINKED ISSUE #1636: Incorrect authority data, the authority set and key gen is `5` but the table displays `1`.
Untitled

### LINKED ISSUE #1637: Unknown green circle and it seems the reputation is still incorrect.
Untitled

### LINKED ISSUE #1655: [BUG] Auto update uptime & reputation without page refresh
### Overview

Authority uptimes & reputation values do not update in real-time without a page refresh even though the values are available. We need to update them in real-time for smooth user experience.

CleanShot 2023-09-20 at 09 31 25

CleanShot 2023-09-20 at 09 30 43

### PR DESCRIPTION
**Pull request by devpavan04**
## Summary of changes

Addresses bugs in `DKG stats-dapp` logged during QA process starting Sept 13, 2023 - #1617 

### Proposed area of change

- [ ] `apps/bridge-dapp`
- [ ] `apps/hubble-stats`
- [x] `apps/stats-dapp`
- [ ] `apps/webbsite`
- [ ] `apps/faucet`
- [ ] `apps/tangle-website`
- [x] `libs/webb-ui-components`

### Reference issue to close (if applicable)

- Closes #1632 
- Closes #1633 
- Closes #1634 
- Closes #1635 
- Closes #1636 
- Closes #1637 
- Closes #1655 
- Closes #1599

### CODE REVIEW

**Comment by drewstone** [comment]
Is this still WIP, what's remaining? Current way PR is described doesn't make it clear. Please use TODOs/checklist and keep it up-to-date.

**Comment by AtelyPham** [comment]
> Is this still WIP, what's remaining? Current way PR is described doesn't make it clear. Please use TODOs/checklist and keep it up-to-date.

I believed this is the checklist Pavan is working on https://github.com/webb-tools/webb-dapp/issues/1617

**Comment by drewstone** [comment]
Sounds good

**Comment by AtelyPham** [comment]
Are there any updates on this PR, @devpavan04? What is the ETA for it?

File: apps/stats-dapp/src/provider/hooks/useReputation.ts
Comment (AtelyPham) [inline apps/stats-dapp/src/provider/hooks/useReputation.ts:1]: Too many `any` in this file @@! Can we minimize `any` usage here?

**Review by AtelyPham — COMMENTED** [review]
Left a comment, otherwise LG 👍

**Pull request merged**